nsv13/code/__DEFINES/overmap.dm

```diff
@@ -27,8 +27,9 @@
 #define FIRE_MODE_RED_LASER 14
 #define FIRE_MODE_BLUE_LASER 15
 #define FIRE_MODE_HYBRID_RAIL 16
+#define FIRE_MODE_LASER_PD 17
```

Comment:
This should probably be inbetween red and blue lasers, so moving it up and shifting the rest down would be better.
Mostly for AI reasons